panel/template/base.py

```diff
@@ -570,6 +579,30 @@ def _get_favicon_type(favicon):
         else:
             raise ValueError("favicon type not supported.")
 
+    @staticmethod
+    def _get_params(value, class_):
+        if isinstance(value, class_):
+            return value
+        if isinstance(value, tuple):
+            value = [*value]
+        elif not isinstance(value, list):
+            value = [value]
+
+        # Important to fx. convert @param.depends functions
+        value = [_panel(item) for item in value]
+
+        if class_==ListLike:
+            return ListLike(objects=value)
+        if class_==GridSpec:
+            grid = GridSpec(ncols=12)
+            for index, item in enumerate(value):
+                grid[index, :]=item
```

Comment:
Suppose this is reasonable, but it is *a bit* weird.

Reply:
This comment I don't understand. What is weird?

If you are referring to the `ncols=12` I've taken that from the ReactTemplate constructor

![image](https://user-images.githubusercontent.com/42288570/97135626-11999600-1751-11eb-8b9f-cf696ddc150b.png)

Reply:
Iterating over the list and assigning each to a row is okay behavior but it's certainly not the only behavior that makes sense. Anyway I'm fine with it.